### PR TITLE
Update SharkRelay Details

### PR DESCRIPTION
**Pull request by mindnervestech**

### CODE REVIEW

File: relayers.json [hunk 1]
```diff
@@ -142,5 +142,24 @@
...
             }
         ]
+    },
+    {
+        "name": "Shark Relay",
+        "url": "https://sharkrelay.com",
+        "networks": [
+            {
+                "networkId": 1,
+                "sra_http_endpoint": "https://api.sharkrelay.com/0x/v0/",
```
Comment (tomhschmidt) [inline relayers.json]: This doesn't appear to be working. Can you verify that this is SRA-compliant? Check out this tool: https://github.com/0xProject/0x-monorepo/tree/development/packages/sra-report

File: relayers.json [hunk 2]
```diff
@@ -142,5 +133,24 @@
...
             }
         ]
+    },
+    {
+        "name": "Shark Relay",
+        "url": "https://sharkrelay.com",
+        "networks": [
+            {
+                "networkId": 1,
+                "sra_http_endpoint": "https://api.sharkrelay.com/api/v0/",
```
Comment (tomhschmidt) [inline relayers.json]: This isn't working for me. Does this comply with the Standard Relayer API?

```
curl https://api.sharkrelay.com/api/v0/token_pairs
<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<title>Error</title>
</head>
<body>
<pre>Cannot GET /api/v0/token_pairs</pre>
</body>
</html>
```

Comment (mindnervestech) [reply relayers.json]: ok. let us check for apis.

Comment (tomhschmidt) [reply relayers.json]: Hey @mindnervestech -- can you run the SRA Report tool on your endpoint and paste the result here? The orderbook endpoint is also failing for me (https://github.com/0xProject/0x-monorepo/tree/development/packages/sra-report)

You can find the full Standard Relayer API spec here 

https://github.com/0xProject/standard-relayer-api/blob/master/README.md

**Pull request merged**